Mage.Sets/src/mage/sets/darksteel/MurderousSpoils.java

```diff
@@ -0,0 +1,123 @@
+ */
+package mage.sets.darksteel;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import mage.ObjectColor;
+import mage.abilities.Ability;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.DestroyTargetEffect;
+import mage.abilities.effects.common.continuous.GainControlTargetEffect;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Rarity;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.CardTypePredicate;
+import mage.filter.predicate.mageobject.ColorPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author wetterlicht
+ */
+public class MurderousSpoils extends CardImpl {
+    
+    private static final FilterCreaturePermanent FILTER = new FilterCreaturePermanent("nonblack creature");
+
+    static {
+        FILTER.add(Predicates.not(new ColorPredicate(ObjectColor.BLACK)));
+    }
+
+    public MurderousSpoils(UUID ownerId) {
+        super(ownerId, 48, "Murderous Spoils", Rarity.UNCOMMON, new CardType[]{CardType.INSTANT}, "{5}{B}");
+        this.expansionSetCode = "DST";
+
+        // Destroy target nonblack creature. It can't be regenerated. You gain control of all Equipment that was attached to it.
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent(FILTER));
+        this.getSpellAbility().addEffect(new MurderousSpoilsEffect());
+        
+    }
+
+    public MurderousSpoils(final MurderousSpoils card) {
+        super(card);
+    }
+
+    @Override
+    public MurderousSpoils copy() {
+        return new MurderousSpoils(this);
+    }
+}
+
+class MurderousSpoilsEffect extends OneShotEffect {
+
+    public MurderousSpoilsEffect() {
+        super(Outcome.DestroyPermanent);
+        staticText = "Destroy target nonblack creature. It can't be regenerated. You gain control of all Equipment that was attached to it.";
+    }
+
+    public MurderousSpoilsEffect(final MurderousSpoilsEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent target = game.getPermanent(source.getFirstTarget());
+        if (target != null) {
+            List<Permanent> attachments = new ArrayList<Permanent>();
+            for (UUID uuid : target.getAttachments()) {
+                Permanent attached = game.getBattlefield().getPermanent(uuid);
+                if (attached.getSubtype().contains("Equipment")) {
+                    attachments.add(attached);
+                }
+            }
+            for (Permanent p : attachments) {
+                ContinuousEffect gainControl = new GainControlTargetEffect(Duration.EndOfGame);

```

Comment:
Use Duration.Custom so the effect is removed if the permanent is no longer on the battlefield